src/modules/dashboard/components/EditColonyDetailsDialog/EditColonyDetailsDialogForm.tsx

```diff
@@ -197,7 +258,12 @@ const EditColonyDetailsDialogForm = ({
           text={{ id: 'button.confirm' }}
           onClick={() => handleSubmit()}
           loading={isSubmitting}
-          disabled={!userHasPermission}
+          disabled={
+            !userHasPermission ||
+            !isValid ||
+            avatarFileError ||
+            !canValuesBeUpdate()
```

Comment:
why this is function? 🤔

Reply:
Custom validation, as we don't have a proper way (using `yup`) of checking if the name _or_ logo changed, but didn't change to the same values they were initially.

It's not ideal. but It's the only way we currently can validate this form properly

Reply:
Hmmm, so why not make it just a variable?

Reply:
Actually a good point.

So initially, this had more "complex" logic and I needed a function to achieve that. But looking at the final result, it is indeed a good candidate to just change into a variable

Reply:
I thought I did? Did I not commit the change? What's going on here????